docs/HowTo/Find-and-Connect/Bootnodes.md

```diff
@@ -1,43 +1,43 @@
-description: Hyperledger Besu Clique Proof-of-Authority (PoA) consensus protocol implementation
-path: blob/master/ethereum/core/src/main/resources/
-source: rinkeby.json
+description: Configuring bootnodoes
 <!--- END of page meta data -->
 
 # Bootnodes
 
-Bootnodes are used to initially discover peers. 
+Bootnodes are used to initially discover peers. A bootnode is a regular node to which nodes connect
+on startup. 
 
 ## Mainnet and Public Testnets
 
 For mainnet, Rinkeby, Ropsten, and Görli, Hyperledger Besu predefines a list of enode URLs.  
 
 ## Private Networks
 
-### Start Bootnode
+In private networks for development or testing purposes, specify one bootnode as described below.
+ 
+In production networks, [configure two or more nodes as bootnodes](../Deploy/Bootnodes.md). 
 
-To start a bootnode for a private network, complete the following steps:
+### Start node to be specified as bootnode 
+
+To start a node to be specified as a bootnode:
 
 1.  Export the public key to a file:
 
     !!! example
         ```bash
-        besu --genesis-file=privateNetworkGenesis.json --data-path=nodeDataPath public-key export --to=bootnode
+        besu --genesis-file=privateNetworkGenesis.json --data-path=nodeDataPath public-key export --to=bootnodePubKey
         ```
         Where `privateNetworkGenesis.json` and `nodeDataPath` are changed to the relevant values for 
         your private network. 
         
-        The node public key is exported to the `bootnode` file.
+        The node public key is exported to the `bootnodePubKey` file.
     
-2. Start the bootnode, specifying:
-
-    * Genesis file and data directory, as in the previous step. 
-    * No arguments for the [`--bootnodes` option](../../Reference/CLI/CLI-Syntax.md#bootnodes) because this is your bootnode.
+2. Start the node specifying the genesis file and data directory. 
     
     !!! example
         ```
-        besu --genesis-file=privateNetworkGenesis.json --data-path=nodeDataPath --bootnodes
-         ```
-     
+        besu --genesis-file=privateNetworkGenesis.json --data-path=nodeDataPath
+        ```
```

Comment:
Optionally add, if using more than 1 bootnode, specify all bootnodes here with `--bootnodes=enode://<publicKeyBootnode1>@<ip1>:30303,enode://<publicKeyBootnode2>@<ip2>:30303,...`

Reply:
But excluding the current bootnode? I don't think it's useful to list its own enode address in the list.

Reply:
AFAIU, it is not useful but it is harmless - so having the same list of bootnodes across all bootnodes might ease deployment. 
If this is right, then I think that it is important to state that listing the node's own address is useless but harmless.

Reply:
Per @EdJoJob and @joshuafernandes we recommend having the same list of bootnodes for all nodes including the bootnodes.

Reply:
Recommend - but, why? What is the purpose of the recommendation? What would happen if not?

Reply:
A recommendation without explanations end up in superstition that might outlive its usefulness

Reply:
Because it's easier to have the same bootnodes list for all nodes that have to remove each node from it's own list.  I'll add a note stating a node ignores itself in the list.

Reply:
So that's what I meant with "useless but harmless". Thank you!